### PR TITLE
enhancement : add analytics tracking to My Bids

### PR DESCRIPTION
**Pull request by xtina-starr**
The type of this PR is: **Enhancement**

This PR introduces the tracking analytics outlined in (https://docs.google.com/spreadsheets/d/1pSVdCEVFERFXCvUqh3mbpJM2BgpdRYosVOCNfcLBccs/edit#gid=0) for My Bids. Currently it implements analytics for the Bids/Inquiries tab, the active/closed lots, and the inbox conversations, tapping on an auction, tapping on Complete Registration CTA, tapping on "Explore auctions" or other "Explore" button (zero state), and screen tracking when the Inbox bottom tab is initially loaded.

---

This PR resolves [PURCHASE-2279]https://artsyproduct.atlassian.net/browse/PURCHASE-2279

### Description

Screen Shot 2020-12-16 at 3 29 22 PM
Screen Shot 2020-12-16 at 3 29 54 PM
Screen Shot 2020-12-16 at 3 31 26 PM
Screen Shot 2020-12-16 at 3 32 33 PM
Screen Shot 2020-12-16 at 3 36 26 PM
Screen Shot 2020-12-17 at 5 13 01 PM
Screen Shot 2020-12-17 at 7 04 02 PM

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [ ] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a [CHANGELOG.yml](/CHANGELOG.yml) entry or my changes do not require one.

[CX-434]: https://artsyproduct.atlassian.net/browse/CX-434
[PURCHASE-2279]: https://artsyproduct.atlassian.net/browse/PURCHASE-2279

### CODE REVIEW

File: src/lib/Scenes/MyBids/MyBids.tsx [hunk 1]
```diff
@@ -86,76 +88,89 @@ class MyBids extends React.Component<MyBidsProps> {
...
-            onRefresh={() => {
-              this.refreshMyBids()
-            }}
-          />
-        }
+      <ProvideScreenTrackingWithCohesionSchema
+        info={{
+          action: ActionType.screen,
+          context_screen_owner_type: OwnerType.inboxBids,
+          // TODO: How to correctly pass the screen that was in view before the Inbox tab was tapped?
```
Comment (ashleyjelks) [inline src/lib/Scenes/MyBids/MyBids.tsx:95]: @ds300 we need to pass the referring screen to this tracking event. Can you suggest a best practice for how to approach this? cc @erikdstock

Comment (ds300) [reply src/lib/Scenes/MyBids/MyBids.tsx:95]: we don't currently store any navigation state history in eigen. Wondering whether the data team can read this info from the previous analytics events? If not we could figure something out I'm sure.

**Review by erikdstock — COMMENTED** [review]
this may be out of date, i forgot to submit days ago

File: src/lib/Containers/Inbox/Inbox.tsx [hunk 2]
```diff
@@ -117,6 +120,20 @@ export class Inbox extends React.Component<Props, State> {
...
+    const tabs = ["inboxBids", "inboxInquiries"]
+
+    return {
+      action: ActionType.tappedNavigationTab,
+      context_module: tabs[index],
+      context_screen_owner_type: tabs[index],
+    }
+  })
+  handleNavigationTab(index: number) {
+    // no op
```
[the remark this answers is not in the document]
Comment (xtina-starr) [reply src/lib/Containers/Inbox/Inbox.tsx:144]: @erikdstock I'm late to the party but class components require the [decorator](https://github.com/artsy/eigen/blob/master/docs/analytics_and_tracking.md#screen-tracking) for tracking.

**Pull request merged**